bundles/binding/org.openhab.binding.zwave/database/zwaveme/zme_rc2.xml

```diff
@@ -0,0 +1,458 @@
+			</Item>
+			<Item>
+				<Value>5</Value>
+				<Label lang="en">Send Preconfigured Scenes</Label>
+			</Item>
+			<Label lang="en">Command to Control Scene No2 (association group 10)</Label>
+			<Help lang="en">This parameter defines the command to be sent to devices of control scene group No2 when the related button is pressed (association group 10)</Help>
+		</Parameter>
+		<Parameter>
+			<Index>19</Index>
+			<Type>list</Type>
+			<Default>4</Default>
+			<Size>1</Size>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">Disabled</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Switch On/Off and Dim (send Basic Set and Switch Multilevel)</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">Switch On/Off only (send Basic Set)</Label>
+			</Item>
+			<Item>
+				<Value>3</Value>
+				<Label lang="en">Switch All</Label>
+			</Item>
+			<Item>
+				<Value>4</Value>
+				<Label lang="en">Send Scenes</Label>
+			</Item>
+			<Item>
+				<Value>5</Value>
+				<Label lang="en">Send Preconfigured Scenes</Label>
+			</Item>
+			<Label lang="en">Command to Control Scene No3 (association group 11)</Label>
+			<Help lang="en">This parameter defines the command to be sent to devices of control scene group No3 when the related button is pressed (association group 11)</Help>
+		</Parameter>
+		<Parameter>
+			<Index>20</Index>
+			<Type>list</Type>
+			<Default>3</Default>
+			<Size>1</Size>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">Disabled</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Switch On/Off and Dim (send Basic Set and Switch Multilevel)</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">Switch On/Off only (send Basic Set)</Label>
+			</Item>
+			<Item>
+				<Value>3</Value>
+				<Label lang="en">Switch All</Label>
+			</Item>
+			<Item>
+				<Value>4</Value>
+				<Label lang="en">Send Scenes</Label>
+			</Item>
+			<Item>
+				<Value>5</Value>
+				<Label lang="en">Send Preconfigured Scenes</Label>
+			</Item>
+			<Label lang="en">Command to Control group All On/Off (association group 12)</Label>
+			<Help lang="en">This parameter defines the command to be sent to devices of control group All On/Off when the related button is pressed (association group 12)</Help>
+		</Parameter>
+		<Parameter>
+			<Index>21</Index>
+			<Type>list</Type>
+			<Default>255</Default>
+			<Size>1</Size>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Switch off only</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">Switch on only</Label>
+			</Item>
+			<Item>
+				<Value>255</Value>
+				<Label lang="en">Switch all on and off</Label>
+			</Item>
+			<Label lang="en">Send the following Switch All commands</Label>
+			<Help lang="en">Send the following Switch All commands</Help>
+		</Parameter>
+	</Configuration>
+
+	<Associations>
+		<Group>
+			<Index>1</Index>
+			<Maximum>5</Maximum>
```

Comment:
This group looks like it should be linked to the controller? If so, can you please add ```<SetToController>true</SetToController>``` in the group? This will soon be used to automatically configure the association group to link to the controller to make things easier for people to configure devices.